internal/storage/storage.go

```diff
@@ -539,6 +556,12 @@ func RemovePackages(r *git.Repository, sourceStage string, packages PackageVersi
 
 // PushChanges function pushes branches to the remote repository.
 func PushChanges(user string, r *git.Repository, stages ...string) error {
```

Comment:
Now that this function depends on a fork, maybe update the godoc comment here to say that?

Reply:
Fixed